corehq/apps/reports/templates/reports/partials/bootstrap5/save_reports_modal.html

```diff
@@ -1,116 +1,172 @@
+              class="form-control vertical-resize"
+              name="description"
+              rows="3"
+              data-bind="value: description"
+            ></textarea>
           </div>
 
-          <div data-bind="visible: has_ucr_datespan">
-            <div class="form-group">  {# todo B5: css:form-group #}
-              <label class="form-label col-sm-3">{% trans "Date Filter" %}</label>
-              <div class="col-sm-9">
-                <select class="form-control"
-                        data-bind="
-                                    options: datespan_filters,
-                                    optionsText: 'display',
-                                    optionsValue: 'slug',
-                                    value: datespan_slug
-                                "></select>
-              </div>
+          <div
+            data-bind="visible: has_ucr_datespan"
+          >
+            <div class="mb-3">
+              <label class="form-label" for="save_report_modal_ucr_datespan">{% trans "Date Filter" %}</label>
+              <select
+                class="form-control"
+                id="save_report_modal_ucr_datespan"
+                name="ucr_datespan"
+                data-bind="
+                  options: datespan_filters,
+                  optionsText: 'display',
+                  optionsValue: 'slug',
+                  value: datespan_slug
+                "
+              ></select>
             </div>
           </div>
 
-          <div data-bind="if: ((!has_ucr_datespan()) && date_range()) || (has_ucr_datespan() && datespan_slug())">
-            <div class="form-group">  {# todo B5: css:form-group #}
-              <label class="form-label col-sm-3" for="date_range">{% trans "Default Date Range" %}</label>
-              <div class="col-sm-9">
-                <select name="date_range" data-bind="value: date_range" class="form-control">
-                  <option value="last7">{% trans "Last 7 days" %}</option>
-                  <option value="last30">{% trans "Last 30 days" %}</option>
-                  <option value="lastmonth">{% trans "Last month" %}</option>
-                  {% if not hide_lastyear %}<option value="lastyear">{% trans "Last year" %}</option>{% endif %}
-                  <option value="lastn">{% trans "Days ago" %}</option>
-                  <option value="since">{% trans "Since a date" %}</option>
-                  <option value="range">{% trans "From a date to a date" %}</option>
-                </select>
-              </div>
+          <div
+            data-bind="if: ((!has_ucr_datespan()) && date_range()) || (has_ucr_datespan() && datespan_slug())"
+          >
+            <div class="mb-3">
+              <label class="form-label" for="save_report_modal_date_range">{% trans "Default Date Range" %}</label>
+              <select
+                id="save_report_modal_date_range"
+                class="form-control"
```

Comment:
This should be `form-select`, shouldn't it?

Reply:
yes, thanks!